inspire/dojson/common/base.py

```diff
@@ -39,6 +39,12 @@ def control_number(self, key, value):
     return value[0]
 
 
+@hep2marc.over('001', 'control_number')
+def control_number2marc(self, key, value):
+    """Record Identifier."""
+    return [value]
```

Comment:
You move the logic from individual functions to `create_record` which might not be desirable for liberal MARC (https://github.com/inveniosoftware/dojson/issues/26).